examples/criteo-example.ipynb

```diff
@@ -166,8 +171,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "train_dataset = nvt.Dataset(train_paths, engine='parquet', part_mem_fraction=0.12)\n",
-    "valid_dataset = nvt.Dataset(valid_paths, engine='parquet', part_mem_fraction=0.12)"
+    "train_dataset = nvt.Dataset(train_paths, engine='parquet', part_mem_fraction=0.15)\n",
+    "valid_dataset = nvt.Dataset(valid_paths, engine='parquet', part_mem_fraction=0.15)"
```

Comment:
This changes performance quite a bit... we can make the change but it will result in like a 23 minute run... All the numbers are calibrated to give us the fastest possible run. Any deviations whether in part_mem_fraction or tree width will result in slower run time.

Reply:
Can we change the defaults inside the library to 0.15 part_mem_fraction and a 4 tree_width? Is there any reason not to do that?

Reply:
the part_mem_fraction is above the limit @rjzamora set out for partitions regularly. This is customized specifically for this data... Other data may perform better  with the current defaults...